addons/logistics_wirecutter/functions/fnc_isFence.sqf

```diff
@@ -20,10 +20,14 @@ params ["_object"];
 TRACE_1("Checking if fence",_object);
 
 private _configOf = configOf _object;
-if !(isNull _configOf) then {
+private _isConfigFence = if !(isNull _configOf) then {
     // Check for isFence entry since we have valid configOf
-    getNumber (_configOf >> QGVAR(isFence)) == 1 // return
+    getNumber (_configOf >> QGVAR(isFence)) == 1
 } else {
-    // Check the p3d name against list (in script_component.hpp)
-    (getModelInfo _object select 0) in FENCE_P3DS // return
+    false
 };
+
+_isConfigFence || {
+    // Check the p3d name against list (in script_component.hpp)
+    (getModelInfo _object select 0) in FENCE_P3DS
+}
```

Comment:
```sqf
getNumber (configOf _object >> QGVAR(isFence)) == 1 || {(getModelInfo _object select 0) in FENCE_P3DS}
```

Reply:
Ah, based off the way it was written I thought that `getNumber` required the config entry to be not null, but now I see it just returns 0 when it's `configNull`. Cool, I'll simplify this bit.